packages/a11y/package.json

```diff
@@ -28,11 +28,25 @@
 	"react-native": "src/index",
 	"types": "build-types",
 	"dependencies": {
-		"@babel/runtime": "^7.16.0",
```

Comment:
The `@babel/runtime` will need to be replaced by `@swc/helpers` and the `jsc.externalHelpers = true` config option. Otherwise all the helpers (for more complex transpilations like classes, async, iterators, ...) would be duplicated in each compiled file.

Reply:
yeah, I saw that, I wanted to check how much of a bundle size impact it would have to avoid the helpers.